packages/scaffolder-test-utils/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@backstage/scaffolder-test-utils",
```

Comment:
Think we should call this `@backstage/plugin-scaffolder-node-test-utils`? Reasoning is that we probably want to introduce frontend test utils at somepoint to be able to help tests `CustomFieldExtensions` and what not and that would mean that this package would have to be isomorphic, which could be a little unfortunate and constrictive.

Reply:
@benjdlambert should it be moved under `plugins/` then?

Reply:
Yep, think so for now at least.